components/footer.jsx

```diff
@@ -4,6 +4,10 @@ import { IntlMixin, FormattedHTMLMessage } from 'react-intl';
 var Footer = React.createClass({
   mixins: [IntlMixin],
   render: function() {
+    var firefox_footer = this.getIntlMessage('firefox_footer');
+    if (window.location.pathname.indexOf('thunderbird') !== -1) {
```

Comment:
Can this be a prop we pass in? Kinda like how we override the message in checkbox now?